core/os/android/adb/commands.go

```diff
@@ -34,7 +38,8 @@ const (
 	ErrDeviceNotRooted = fault.Const("Device is not rooted")
 	ErrRootFailed      = fault.Const("Device failed to switch to root")
 
-	maxRootAttempts = 5
+	maxRootAttempts                            = 5
+	GPURENDERINGSTAGESDATASOURCEDESCRIPTORNAME = "gpu.renderstages"
```

Comment:
Use camel-case and I think it should start with a lowercase letter (means that it's private).